lib/special_route_publisher.rb

```diff
@@ -81,6 +81,14 @@ def routes
         title: "Prepare your business for the UK leaving the EU",
         description: "Q&A frontend for the 'Find EU Exit guidance for your business' finder",
         type: "exact",
+      },
+      {
+        rendering_app: "finder-frontend",
+        base_path: "/uk-nationals-living-in-the-eu",
+        content_id: "????",
```

Comment:
^^ how would I get a content ID for this page?

Reply:
You can generate one yourself, `SecureRandom.uuid` should give you one in a Ruby console.

Reply:
Thanks ! updated